tool.ts

```diff
@@ -57,6 +60,15 @@ export function isExplicitVersion(versionSpec: string) {
 }
 
 /**
+ * Returns cleaned (removed leading/trailing whitespace, remove '=v' prefix)
+ * and parsed version, or null if version is invalid.
+ */
+export function cleanVersion(version: string) {
```

Comment:
exposing clean() is needed to fix a bug in nodeTool ([PR here](https://github.com/Microsoft/vsts-tasks/pull/3909/files))